dash_app/dashboard.py

```diff
@@ -102,14 +92,32 @@ def init(server):
     )
     agg_has_timeseries = _agg_wide_var_counts(wide_var_has_timeseries).reset_index()
 
-    wide_var_has_mask = ds.tag.loc[:, :, TagType.SOURCE_URL].unstack(PdFields.VARIABLE).notnull()
-    agg_has_url = _agg_wide_var_counts(wide_var_has_mask.astype(int)).reset_index()
+    wide_var_has_url = ds.tag.loc[:, :, TagType.SOURCE_URL].unstack(PdFields.VARIABLE).notnull()
+    agg_has_url = _agg_wide_var_counts(wide_var_has_url.astype(int)).reset_index()
 
     counties = ds.get_subset(aggregation_level=AggregationLevel.COUNTY)
     # TODO(tom): update to work with Source tag
     has_url = counties.tag.loc[:, :, TagType.SOURCE_URL].unstack(PdFields.VARIABLE).notnull()
     county_variable_population_ratio = population_ratio_by_variable(counties, has_url)
 
+    # A table of regions in a DataFrame.
+    static_columns = pd.Index(
+        [CommonFields.LOCATION_ID, CommonFields.COUNTY, CommonFields.STATE, CommonFields.POPULATION]
+    ).intersection(ds.static.columns)
+    df_regions = ds.static.loc[:, static_columns].copy()
```

Comment:
seems like this function will be growing in size, might be good to pull the `df_regions` generation logic into it's own function

Reply:
I tried to got a step further with a032a38 and created a dataclass to hold all the per region variables.